actors/miner/src/monies.rs

```diff
@@ -183,10 +183,11 @@ pub fn pledge_penalty_for_termination(
     reward_estimate: &FilterEstimate,
     replaced_day_reward: &TokenAmount,
     replaced_sector_age: ChainEpoch,
+    current_extension_age: ChainEpoch,
```

Comment:
Could you document this parameter? It's not obvious to me where it should come from. From usage, I'm guessing its intended as the period of the duration commitment that has been satisfied? Or period for which reward_estimate is accurate? (Are they always the same?). Is it necessarily about extension? What about snapdeals that changes the reward?